test/api-digester/stability-stdlib-abi-with-asserts.test

```diff
@@ -38,7 +38,8 @@
 // Thank you for your help ensuring the stdlib remains compatible with its past!
 //                                            -- Your friendly stdlib engineers
 
-// REQUIRES: swift_stdlib_asserts, array_cow_checks
+// REQUIRES: swift_stdlib_asserts
+// UNSUPPORTED: array_cow_checks
```

Comment:
Note: This will need to be reverted before this lands -- we're working on cleaning this up in https://github.com/apple/swift/pull/41045.

Reply:
fantastic, thanks!